programs/timefn.c

```diff
@@ -8,6 +8,7 @@
  * You may select, at your option, one of the above-listed licenses.
  */
 
+#define _POSIX_C_SOURCE	200809L /* clock_gettime */
```

Comment:
I believe this should rather be defined in `timefn.h`, just before the `#include <time.h>`,
otherwise, `UTIL_time_t` declaration in `timefn.h` will be out of sync with `timefn.c`.